paper-icon-item.html

```diff
@@ -33,10 +33,16 @@
 
 The following custom properties and mixins are available for styling:
 
-Custom property | Description | Default
-----------------|-------------|----------
-`--paper-item-icon-width` | Width of the icon area     | `56px`
-`--paper-icon-item`       | Mixin applied to the item  | `{}`
+Custom property               | Description                                    | Default
+------------------------------|------------------------------------------------|----------
+`--paper-item-icon-width`     | Width of the icon area                         | `56px`
+`--paper-icon-item`           | Mixin applied to the item                      | `{}`
+`--paper-item-selected-weight`| The font weight of a selected item             | `bold`
+`--paper-item-selected`       | Applied to selected paper-items                | `{}`
```

Comment:
(for consistency) Mixin applied to...

(also below you use "A mixin applied" and above just "Mixin applied" and maybe in the same block they should be the same? :smile: )